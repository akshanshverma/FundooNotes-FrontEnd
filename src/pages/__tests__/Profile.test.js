import React from 'react';
import { shallow } from 'enzyme';
import Profile from "../Profile";


const component = shallow(<Profile history={[]} debug />);
describe('Profile', () => {
    it('should render correctly in "debug" mode Profile', () => {
        expect(component).toMatchSnapshot();
    });
});