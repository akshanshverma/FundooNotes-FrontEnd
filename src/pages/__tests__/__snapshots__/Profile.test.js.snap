// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile should render correctly in "debug" mode Profile 1`] = `
<div
  className="maindiv"
>
  <WithStyles(Card)
    id="card"
  >
    <div
      className="userData"
    >
      <div
        className="username"
      >
        username: 
      </div>
      <div
        className="email"
      >
        email: 
      </div>
    </div>
    <div
      className="logoutButton"
    >
      <WithStyles(Button)
        color="primary"
        id="logout"
        onClick={[Function]}
        type="Submit"
        variant="contained"
      >
        logout
      </WithStyles(Button)>
    </div>
  </WithStyles(Card)>
</div>
`;
